model/src/main/java/com/expediagroup/streamplatform/streamregistry/model/ClusterKey.java

```diff
@@ -12,13 +13,18 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.expediagroup.streamplatform.streamregistry.exceptions;
+package com.expediagroup.streamplatform.streamregistry.model;
```

Comment:
Weird how git registered this as a move from `ActorNotFoundException` to `ClusterKey`